src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -996,19 +997,31 @@ private static void cleanUpGlobalClassValue(@Nonnull ClassLoader loader) throws
         Object map = mapF.get(globalClassValue);
         Class<?> groovyClassValuePreJava7Map = Class.forName("org.codehaus.groovy.reflection.GroovyClassValuePreJava7$GroovyClassValuePreJava7Map");
         Collection entries = (Collection) groovyClassValuePreJava7Map.getMethod("values").invoke(map);
-        Field klazzF = classInfoC.getDeclaredField("klazz");
-        klazzF.setAccessible(true);
         Method removeM = groovyClassValuePreJava7Map.getMethod("remove", Object.class);
```

Comment:
Could be a nice refinement to cleanup code later. In this case the number of accesses is fairly small so the cost is probably insignificant compared to other aspects of the build. Anyway out of scope for this PR.